snapshots/testsuite/testsuite.go

```diff
@@ -879,6 +885,8 @@ func checkRootPermission(ctx context.Context, t *testing.T, snapshotter snapshot
 
 func check128LayersMount(name string) func(ctx context.Context, t *testing.T, snapshotter snapshots.Snapshotter, work string) {
 	return func(ctx context.Context, t *testing.T, snapshotter snapshots.Snapshotter, work string) {
+		// While we don't test Aufs on CI anymore and Aufs snapshotter will be deprecated in future,
+		// we will keep this if condition for people who use Aufs.
```

Comment:
Is it fair to say we will deprecate aufs snapshotter? It is not in the mainline and overlayfs is essentially fulfilling the needs.